bin/README.md

```diff
@@ -6,8 +6,9 @@ bencharmking purposes.
 ## Running
 
 ```
+npm install -g typescript
```

Comment:
Why is a global install of TypeScript needed to run the benchmarks?

Reply:
It's needed for `npm install` to work.

Reply:
It shouldn't be needed.. What error did you receive?

Reply:
Tried to repro but couldn't. Closing.